tests/StaffController.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('StaffController', function () {
+	var $controller;
+
+	beforeEach(module('app'));
+
+	beforeEach(inject(function (_$controller_) {
+		$controller = _$controller_;
+	}));
+
+
+	it('should not be using $scope to set values', function () {
```

Comment:
these tests are really nice - it will be clear to the student why it's failing and how they can fix it